generate-release-vars.sh

```diff
@@ -49,6 +49,10 @@ ami_id_al2_x86=$(aws ssm get-parameters --region "$region" --names /aws/service/
 ami_name_al2_x86=$(aws ec2 describe-images --region "$region" --owner amazon --image-id "$ami_id_al2_x86" --query 'Images[0].Name' --output text)
 ami_id_al2_arm=$(aws ssm get-parameters --region "$region" --names /aws/service/ami-amazon-linux-latest/amzn2-ami-minimal-hvm-arm64-ebs --query 'Parameters[0].[Value]' --output text)
 ami_name_al2_arm=$(aws ec2 describe-images --region "$region" --owner amazon --image-id "$ami_id_al2_arm" --query 'Images[0].Name' --output text)
+ami_id_al2_kernel5dot10=$(aws ssm get-parameters --region "$region" --names /aws/service/ami-amazon-linux-latest/amzn2-ami-minimal-hvm-x86_64-ebs --query 'Parameters[0].[Value]' --output text)
```

Comment:
since we're only the same AL2 base AMI as we do today, could we just refer to that instead and avoid creating these? you can also add a comment to indicate that both 4.x and 5.x ECS AMIs use the same base AMI, we just do kernel upgrade during AMI build.

Reply:
I previously had this thought as well, but in the future when Amazon Linux releases their minimal AL2 kernel 5.10 base AMI, we will migrate to use that base AMI instead of the current one. As a result, I decided to create and keep these variables so that when we do migrate, we can simply just change these variables to point to the new minimal AL2 kernel 5.10 base AMIs. That is, regardless of what we decide to do now, we will eventually need to create these new variables in the future anyway.